dockerfiles/e2e-saas/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+FROM quay.io/kkanova/upstream_e2e
```

Comment:
what  is the base image about? and why do we need some custom personal e2e upstream image? @rhopp this looks like a hacky approach for me - what is the long-running plan/approach for e2e adoption in downstream?

Reply:
Now the image is based on upstream docker image.